app/commands/v2/represent_downstream.rb

```diff
@@ -48,7 +48,7 @@ def downstream_live(content_id, locale)
             content_id: content_id,
             locale: locale,
             payload_version: event.id,
-            message_queue_update_type: "links",
+            message_queue_event_type: "links",
```

Comment:
I just wanted to avoid making a breaking change in this PR, I'll speak to Search and see what they think.